cmd/buildah/push.go

```
@@ -78,15 +79,23 @@ var (
 )
 
 func pushCmd(c *cli.Context) error {
-	args := c.Args()
-	if len(args) < 2 {
-		return errors.New("source and destination image IDs must be specified")
-	}
 	if err := parse.ValidateFlags(c, pushFlags); err != nil {
 		return err
 	}
-	src := args[0]
-	destSpec := args[1]
+
+	args := c.Args()
+	var src, destSpec string
+	switch len(args) {
+	case 0:
+		return errors.New("source and destination image IDs must be specified")
```

Should probably throw an error if the user specifies 3 or more Args.